### PR TITLE
maria.nistorescu -project documentation

### PR DESCRIPTION
**Pull request by LorenaNistorescu**
### Pull Request Overview

This pull request is about project documenation.

### Author

Signed-off-by: Nistorescu Maria-Lorena <evelinsmith289@gmail.com>

### CODE REVIEW

**Comment by alexandruradovici** [comment]
@LorenaNistorescu please fix this so we can merge it. Rename the folder to your username and the documentation to *index.md*.

**Comment by alexandruradovici** [comment]
Move the cicada files to your project's repository.

File: docs/project/2024/Lorena Nistorescu/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Smart Door Lock System
+Smart door lock: Keyless security for your space.
+
```
Comment (alexandruradovici) [inline docs/project/2024/Lorena Nistorescu/index.md:3]:
```suggestion
:::info
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Pleaee rename your folder to the username from upb and fill the pull request template.

**Comment by LorenaNistorescu** [comment]
I am trying, but honestly I can't find to change the folder of my name from
Lorena Nistorescu to maria.nistorescu...

În dum., 19 mai 2024 la 23:20, Alexandru Radovici ***@***.***>
a scris:

**Comment by LorenaNistorescu** [comment]
[image: image.png]Am ambele branch-uri aici, numele la folder nu il pot
schimba, nu stiu de ce nu merge editat..?

În dum., 19 mai 2024 la 23:27, Evelin Smith ***@***.***> a
scris:

**Comment by alexandruradovici** [comment]
Clone the repo, change them locally, commit the chanfes and push them.

File: docs/project/2024/Lorena Nistorescu/DIAGRAMdoor.png
Comment (alexandruradovici) [inline docs/project/2024/Lorena Nistorescu/DIAGRAMdoor.png:1]: The folder name is still wrong, please use your username.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Pull request closed without merging**